src/ui/map.js

```diff
@@ -192,9 +192,9 @@ class Map extends Camera {
 
         bindHandlers(this, options);
 
-        this._hash = options.hash && (new Hash()).addTo(this);
+        this.hash = options.hash && (new Hash()).addTo(this);
```

Comment:
IMO we should keep the `_` to signal that this is a private member rather than a public API.